recipes/r-rmysql/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rmysql
+  # Note that conda versions cannot contain -, so any -'s in the version have
+  # been replaced with _'s.
+  version: "0.10.9"
+
+source:
+  fn: RMySQL_0.10.9.tar.gz
+  url:
+    - http://cran.r-project.org/src/contrib/RMySQL_0.10.9.tar.gz
+    - http://cran.r-project.org/src/contrib/Archive/RMySQL/RMySQL_0.10.9.tar.gz
+  md5: 3628200a1864ac3005cfd55cc7cde17a
+
+
+  # You can add a hash for the file here, like md5 or sha1
+  # md5: 49448ba4863157652311cc5ea4fea3ea
+  # sha1: 3bcfbee008276084cbb37a2b453963c61176a322
+  # patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
```

Comment:
Also feel free to remove these comments and the above. Everything what is not needed :)